### PR TITLE
Finishing version 1

### PR DESCRIPTION
**Pull request by j-stri**

### CODE REVIEW

File: src/View.elm [hunk 1]
```diff
@@ -105,11 +105,25 @@ rightPane model =
...
-        Lost { nextPiece } ->
-            viewNextPiece nextPiece
+        Lost { nextPiece, score } ->
+            Svg.g []
+                [ viewNextPiece nextPiece
+                , viewScore score
+                ]
+
+
+viewScore score =
```
Comment (miniBill) [inline src/View.elm:121]: Add type signature (`Int -> Svg msg`)

File: src/View.elm [hunk 2]
```diff
@@ -247,13 +261,27 @@ viewLost _ =
...
+    let
+        xcoord = coordinate (screenWidth / 2)
+        ycoord = coordinate (screenHeight / 2)
+
+        width = 25
+
+        height = 2
+
+
+    in
```
Comment (miniBill) [inline src/View.elm:273]: Add type annotations

File: src/View.elm [hunk 3]
```diff
@@ -247,13 +261,27 @@ viewLost _ =
...
+    
+    Svg.g []
+        [ rect "gray" (screenWidth / 2 - width / 2) (screenHeight / 2 - height / 2 - 0.12) width height
+        , text_
+            [ textAnchor "middle"
+            , alignmentBaseline "middle"
+            , x <| xcoord
+            , y <| ycoord
```
Comment (miniBill) [inline src/View.elm:281]: The `<|`s are redundant here

**Review by miniBill — CHANGES_REQUESTED** [review]
Format all files, observe the comments, in general the code is fine

File: src/View.elm (hunk 3 above)
Comment (Moonlit36) [reply src/View.elm]: I removed them is that ok?

**Pull request merged**